src/Microsoft.Diagnostics.Monitoring.EventPipe/Counters/ICountersLogger.cs

```diff
@@ -0,0 +1,21 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Collections.Generic;
+using System.Text;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace Microsoft.Diagnostics.Monitoring.EventPipe
+{
+    internal interface ICountersLogger
+    {
+        //TODO Consider making these async.
```

Comment:
Should you do this now?

Reply:
I will push this out as a post checkin issue.